cmd/zfs/zfs_main.c

```diff
@@ -4721,8 +4784,17 @@ zfs_do_send(int argc, char **argv)
 	if (flags.replicate && fromname == NULL)
 		flags.doall = B_TRUE;
 
-	err = zfs_send(zhp, fromname, toname, &flags, STDOUT_FILENO, NULL, 0,
-	    flags.verbosity >= 3 ? &dbgnv : NULL);
+	err = zfs_send(zhp, fromname, toname, &flags, STDOUT_FILENO,
+	    exclude_context.count > 0 ? zfs_do_send_exclude : NULL,
+	    &exclude_context, flags.verbosity >= 3 ? &dbgnv : NULL);
+
+	if (exclude_context.count > 0) {
+		/* Let's free it all up */
+		size_t i;
+		for (i = 0; i < exclude_context.count; i++)
```

Comment:
```suggestion
		for (size_t i = 0; i < exclude_context.count; i++)
```

Reply:
Switching back and forth between 3 different coding styles has confused me. ;)